### DISCUSSION TITLE
Apextree

### DISCUSSION THREAD

**Discussion by scoldwell** [apexcharts/apexcharts.js#4449] [General]
Where is the repository for apextree? I have searched for it and come up empty.

**Comment by hvsharma63** [comment]
@scoldwell I am looking for the same.

**Reply by hvsharma63** [reply]
@junedchhipa

**Comment by jaguarman** [comment]
same here, where is it?

**Comment by devcaeg** [comment]
👀

**Reply by pauloffb** [reply]
https://github.com/apexcharts/apextree
